src/main.cpp

```diff
@@ -2956,13 +2976,27 @@ bool Session::SeekTime(double seekTime, unsigned int streamId, bool preceeding)
         (streamId == 0 || (*b)->info_.GetPhysicalIndex() == streamId))
     {
       bool bReset;
+      // all streams must be started before seeking to ensure cross chapter seeks
```

Comment:
Not for this PR but variables called `b` and `e` are not easy to follow. Maybe upon a larger refactor some of these can be fixed.

Reply:
Yes definitely agreed, particularly this nightmare from here onwards: https://github.com/xbmc/inputstream.adaptive/blob/1247d4bcff218db54c3c2ec30be020913731f09d/src/parser/DASHTree.cpp#L1664